### ISSUE TITLE
Json output is incorrect

### ISSUE THREAD

**Issue by ashcoding** [davglass/license-checker#64]
Here's an example of the Json output
```json
  "\u001b[34mdate-now\u001b[39m\u001b[2m@\u001b[22m\u001b[32m0.1.4\u001b[39m": {
    "licenses": "MIT",
    "repository": "https://github.com/Colingo/date-now",
    "licenseFile": "/node_modules/date-now/LICENCE"
  },
```
The name has the color included as well. This should be removed.

**Comment by davglass** [comment]
Published in `license-checker@5.1.1`

**Comment by ashcoding** [comment]
:+1: Thank you! That's fast. :)
